rest/taskana-rest-spring-example-boot/src/main/java/pro/taskana/rest/ExampleRestConfiguration.java

```diff
@@ -31,4 +33,11 @@ public PlatformTransactionManager txManager(DataSource dataSource) {
   public SampleDataGenerator generateSampleData(DataSource dataSource) {
     return new SampleDataGenerator(dataSource, schemaName);
   }
+
+  @Bean(initMethod = "start", destroyMethod = "stop")
```

Comment:
maybe we should make a note that this is only required to let the adapter example connect to the same database? 

And I would you another port as 9090. this is still a rather common port for web applications.